src/Files.App/Actions/Navigation/NavigateBackAction.cs

```diff
@@ -22,8 +22,8 @@ public HotKey SecondHotKey
 		public HotKey ThirdHotKey
 			=> new(Keys.Mouse4);
 
-		public HotKey MediaHotKey
-			=> new(Keys.GoBack, false);
+		//public HotKey MediaHotKey
+		//	=> new(Keys.GoBack, false);
```

Comment:
Why is this commented out?

Reply:
Because I had decided not to support hidden keys.
But I will revert them.

Reply:
Done